src/app/personside/MainLayoutStyles.tsx

```diff
@@ -20,24 +19,51 @@ interface StyleProps {
     dialogPanelEkspandert?: boolean;
 }
 export const VenstreKolonne = styled.section<StyleProps>`
-    width: ${props => (props.dialogPanelEkspandert ? '50%' : '70%')};
+    ${props =>
+        props.dialogPanelEkspandert
+            ? css`
+                  flex: 70% 1 1;
+              `
+            : css`
+                  flex: 90% 1 1;
+              `};
     padding: ${theme.margin.layout};
     display: flex;
     flex-direction: column;
+    overflow-y: scroll;
     > * {
         border-radius: ${theme.borderRadius.layout};
     }
 `;
 
 export const HøyreKolonne = styled.section<StyleProps>`
-    width: ${props => (props.dialogPanelEkspandert ? '50%' : '30%')};
-    background-color: white;
+    ${props =>
+        props.dialogPanelEkspandert
+            ? css`
+                  flex: 30% 1 1;
+              `
+            : css`
+                  justify-content: flex-end;
+                  > *:not(:last-child) {
+                      display: none;
+                  }
+                  &:before {
+                      content: 'Dialogpanel';
```

Comment:
Hvordan blir det her med en skjermleser?

Reply:
Skjermleseren leser det opp. Men bør nok likevel jobbes litt med denne minimerinsløsningen for at det skal funke optimalt både med og uten skjermleser. Se på det som work in progress ;-)